andela/wg-millennium-falcon
# #157731566: Show all infos for an exercise

### What does this PR do?
This PR enables an sms and telegram alert to be sent to a user to let them know about their check status.

### Description of Task to be completed?
There should be a new read-only API endpoint that collects all the information of an exercise such as muscles, category, images, etc. This makes it easier for clients to get all the information since only a single request is needed to be done

### How should this be manually tested?
* Run the application
* Navigate to the restAPI root 
* click on the api/v2/exerciseinfo route
* This should return all the exercises and the respective information in json format.
<img width="755" alt="screen shot 2018-05-31 at 10 51 49" src="https://user-images.githubusercontent.com/26734746/40769502-d11993c2-64c0-11e8-8766-0c3bdd9aac4f.png">


### What are the relevant pivotal tracker stories?
[#157731566](https://www.pivotaltracker.com/story/show/157731566)
